src/external/aabbcc/AABB.cc

```diff
@@ -403,7 +403,12 @@ namespace aabb
         unsigned int node = allocateNode();
 
         // AABB size in each dimension.
+        // GCC supports variable length arrays, but MSVC does not.
+#ifdef _MSC_VER
+        std::vector<double> size(dimension);
+#else
         double size[dimension];
```

Comment:
This is likely faster because it would mean that a larger stack is allocated. I don't just exchange this with a vector as a vector means heap allocation.